browser_history/browsers.py

```diff
@@ -91,6 +119,79 @@ def _deeper(array, folder, bookmarks_list):
                     )
         return bookmarks_list
 
+    def cookies_parser(self, cookie):
+
+        """Returns cookies of a single input for Chrome based browsers
+        The returned datetimes are timezone-aware with the local timezone set
+        by default
+
+        :param cookie : dictionary containing cookie details
+        :type cookie : dict
+        :return a tuple of cookie information
+        :rtype: list(tuple(:py:class: str, str, `datetime.datetime`, str, str, str))
+        """
+        platform = utils.get_platform()
+        if EXC is not None:
+            logger_map = {
+                utils.Platform.LINUX: "'keyring', 'pbkdf2', 'pyaes', 'gi'. ",
+                utils.Platform.WINDOWS: "win32crypt. ",
+                utils.Platform.MAC: "'keyring', 'pbkdf2'. ",
+            }
+            utils.logger.info(
+                "%s",
+                "Module Not Found: "
+                + "The following packages are required for retrieving"
+                + " cookies on Chrome-based browsers:"
+                + logger_map[platform]
+                + "Install using `pip install -r cookie-requirements.txt`.",
```

Comment:
The `cookie-requirements.txt` wouldn't be available to users who installed browser-history through pip. You can drop that line.
Also, we could make this whole string as the error message instead of just `EXC`.
Also, rename `EXC` to something else. Names in all caps usually means constants in python.

Reply:
For some reason pylint considers all global variables as contants xD
Thats why had to make it capital

> The `cookie-requirements.txt` wouldn't be available to users who installed browser-history through pip. You can drop that line.

Good catch xD

Reply:
> For some reason pylint considers all global variables as contants xD

Oh lol. Maybe make the cookie import part into a function?

Reply:
But wont it look odd in browsers.py whose main objective is to have browsers only

Reply:
Also since both bookmarks_parser and cookies_parser will soon moved to another class in another file

Reply:
Good points. For now, you can keep it there and when the parsers are moved to a different file, you can move it there. (also add a comment before the imports indicating what those are for)